spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1416,13 +1423,20 @@ bom {
 			]
 		}
 	}
-	library("Rabbit AMQP Client", "5.13.0") {
+	library("RabbitMQ AMQP Client", "5.13.0") {
 		group("com.rabbitmq") {
 			modules = [
 				"amqp-client"
 			]
 		}
 	}
+	library("RabbitMQ Stream Client", "0.2.0-SNAPSHOT") {
```

Comment:
Is there an ETA for this having a tagged release? We don't want to be on a snapshot for too long and also wouldn't want to revert if the release isn't available before our next milestone.

Reply:
I will ask them when it is expected to be released, spring-rabbit-stream 2.4.0-M1 uses 0.1.0 (not a snapshot), but this needs a new feature in 0.2.0.

Reply:
@wilkinsona It will be released sometime next week.

https://pivotal.slack.com/archives/CSC92F50B/p1627071597374400?thread_ts=1627070500.374200&cid=CSC92F50B

Reply:
Excellent. Given that it's not long, let's wait till the release rather than adding the Sonatype snapshot repo.